lib/utils/mbaas-token-check.js

```diff
@@ -1,23 +1,31 @@
-var _ = require('underscore');
-var common = require("../common");
-var fhreq = require("./request");
+const _ = require('underscore');
```

Comment:
I think we need to be careful here as not all customers will be running node 4 +